### DISCUSSION TITLE
Talks & Presentations

### DISCUSSION THREAD

**Discussion by ConorOkus** [lightningdevkit/rust-lightning#1557] [General]
Here is a place you can share your talks/panels/presentations/slides with the LDK community.

**Comment by arik-so** [comment]
Here are the slides for @jkczyz's and my [presentation](https://youtu.be/9-81tobFSKg) at btc++ Austin 2022.

[LDK.pdf](https://github.com/lightningdevkit/rust-lightning/files/8952209/LDK.pdf)

**Comment by ConorOkus** [comment]
Here is the [talk](https://www.youtube.com/watch?v=l3OkLI_nT8k&list=WL&index=25) from [johncantrell97](https://github.com/johncantrell97) at btc++ on [Sensei](https://l2.technology/sensei).

**Comment by ConorOkus** [comment]
Ryan Loomba's [presentation](https://www.youtube.com/watch?v=5YmfEgh-LC8&t=2442s) at btc++ explaining some of the trade-offs their team had to think about when comparing LDK to other solutions.

**Comment by ConorOkus** [comment]
Danny Diekroeger software engineer at CashApp talked about LDK on Kevin Rooke's podcast. Twitter summary [here](https://twitter.com/lightningdevkit/status/1564350441177157632)
